internal/entimport/import.go

```diff
@@ -203,3 +205,163 @@ func isJoinTable(table *schema.Table) bool {
 	}
 	return true
 }
+
+func typeName(tableName string) string {
+	return inflect.Camelize(inflect.Singularize(tableName))
+}
+
+func tableName(typeName string) string {
+	return inflect.Underscore(inflect.Pluralize(typeName))
+}
+
```

Comment:
Those two are not "bijective". 

`user_name` => `UserName` => `user_names`

Reply:
This one a one-way tool, sql->atlas->ent schema.
This funcs are used for different purposes and without any relation to each other.